biz.aQute.launchpad.tests/test/aQute/xlaunchpad/LaunchpadConfigurationTest.java

```diff
@@ -0,0 +1,59 @@
+package aQute.xlaunchpad;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.osgi.framework.Bundle;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import aQute.launchpad.Launchpad;
+import aQute.launchpad.LaunchpadBuilder;
+import aQute.launchpad.Service;
+import aQute.launchpad.junit.LaunchpadRunner;
+
+@SuppressWarnings("restriction")
+@RunWith(LaunchpadRunner.class)
+public class LaunchpadConfigurationTest {
+
+	static LaunchpadBuilder builder = new LaunchpadBuilder().snapshot()
+		.bndrun("runsystempackages.bndrun")
+		.runfw("jar/org.apache.felix.framework-6.0.2.jar;version=file")
+		.export("*")
+		.bundles(
+			"org.osgi.util.promise, org.osgi.util.function, jar/org.apache.felix.scr-2.1.16.jar;version=file, jar/org.apache.felix.configadmin-1.9.8.jar;version=file, jar/org.apache.felix.configurator-1.0.8.jar;version=file, org.assertj.core")
```

Comment:
Hallo Jürgen,

is it necessary to use fixed versions of framework/ca/scr  and to the one in the /jar folder?

see.
https://github.com/bndtools/bnd/blob/master/biz.aQute.launchpad.tests/test/aQute/xlaunchpad/Manual.java

Reply:
I just oriented myself on the existing test, that only used bundles from the jars in the test itself. Thus I think this is okay.